src/ICS20Transfer.sol

```diff
@@ -170,6 +134,49 @@ contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, Reentr
         emit ICS20Timeout(packetData);
     }
 
+    /// @notice Create a MsgSendPacket for an ICS20 transfer
+    /// @notice This function is meant as a helper function to easily construct a correct MsgSendPacket
+    /// @param msg_ The message for sending a transfer
+    /// @return The constructed MsgSendPacket
+    function createMsgSendPacket(SendTransferMsg calldata msg_)
+        public
+        view
```

Comment:
What do we need to do to convert this to a pure function? I guess contracts that call others cannot be pure?

Reply:
This is only doing a library call, so might be possible if the library call was pure. Not sure if static call helps with that or not, I assume not?